src/index.tsx

```diff
@@ -9,33 +9,38 @@ import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
 //  see https://github.com/microsoft/fluentui/wiki/Using-icons
 initializeIcons();
 
-if (import.meta.env.DEV) {
-  const browser = await import("./mocks/browser.js" as any);
-  browser.worker.start({
-    onUnhandledRequest(
-      req: { url: { pathname: string } },
-      print: { warning: () => void }
-    ) {
-      if (
-        req.url.pathname.startsWith("/favicon.ico") ||
-        req.url.pathname.startsWith("/manifest.json") ||
-        req.url.pathname.endsWith(".png") // Ignore giving warning for things like the logo, the persona icons, etc
+async function prepareMSW() {
+  if (import.meta.env.DEV) {
```

Comment:
When we do a build test, and set the mode to testing, I assume that this is still flagged as DEV, and would add MSW into our bundle, which wouldn't be desired, correct?